custom/icds/location_rationalization/forms.py

```diff
@@ -44,3 +45,28 @@ def _validate_file(self, uploaded_file):
             raise forms.ValidationError(_("Missing following columns in sheet: {columns}").format(
                 columns=", ".join(missing_headers)
             ))
+
+
+class LocationRationalizationRequestTemplateForm(forms.Form):
+    location_id = forms.CharField(label=ugettext_lazy("Location"), widget=forms.widgets.Select(choices=[]),
+                                   required=True)
+    location_type = forms.ChoiceField(label=ugettext_lazy("Location Type"), choices=(), required=True,
+                                      help_text=_("the location type each row should represent in download"
```

Comment:
I don't understand this. If I specify "block" here, do I get blocks and lower locations, or blocks and higher locations?

Reply:
block and higher locations. basically these needs the leaf node or the lower most location type. AWC is case of icds.

Reply:
Will ICDS always choose AWC? Or is the idea that if you aren't making any AWC-level changes, you can choose a higher level and then your file is shorter?

Reply:
...I think I answered my own question & understand this now.

Reply:
ICDS would always choose AWC. Since we can't determine what the leaf node would be, since there can be multiple, we want the user to tell us.